freeipa/src/main/java/com/sequenceiq/freeipa/init/ModifyAdminPassordExpiration.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.freeipa.init;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.core.task.AsyncTaskExecutor;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.logger.MDCBuilder;
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.entity.Stack;
+import com.sequenceiq.freeipa.service.freeipa.FreeIpaClientFactory;
+import com.sequenceiq.freeipa.service.freeipa.user.UserService;
+import com.sequenceiq.freeipa.service.stack.StackService;
+
+@Component
+public class ModifyAdminPassordExpiration implements ApplicationListener<ContextRefreshedEvent> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ModifyAdminPassordExpiration.class);
+
+    @Inject
+    private StackService stackService;
+
+    @Inject
+    private FreeIpaClientFactory freeIpaClientFactory;
+
+    @Inject
+    private AsyncTaskExecutor asyncTaskExecutor;
+
+    @Inject
+    private UserService userService;
+
+    @Override
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        asyncTaskExecutor.submit(this::modifyPasswordExpiratiomForAllStack);
+    }
+
+    private void modifyPasswordExpiratiomForAllStack() {
```

Comment:
typo in method name: Expiratiom -> Expiration

Reply:
thanks, fixed it